db/migrate/20150618112012_add_old_route_to_tags.rb

```diff
@@ -0,0 +1,10 @@
+class AddOldRouteToTags < ActiveRecord::Migration
+  def up
+    add_column :tags, :has_legacy_route_without_namespace, :boolean
+    execute 'UPDATE tags SET has_legacy_route_without_namespace = true WHERE state = "published"'
```

Comment:
This update needs to be constrained to tags of type Topic.  We don't want to set this for browse pages.